tokio/src/runtime/mod.rs

```diff
@@ -334,7 +335,11 @@ impl Runtime {
     /// [threaded scheduler]: index.html#threaded-scheduler
     /// [basic scheduler]: index.html#basic-scheduler
     /// [runtime builder]: crate::runtime::Builder
-    pub fn new() -> io::Result<Self> {
+    pub fn new() -> io::Result<Runtime> {
+        fn bounds<T: Send + Sync>() {}
```

Comment:
This should probably be placed in its own test rather than inline in `new` like this.